robot_skills/src/robot_skills/arms.py

```diff
@@ -22,6 +22,50 @@
 # Temporary: this should be approximately 0.02 once the sensor is correctly setup
 GRASP_SENSOR_LIMITS = tuple(rospy.get_param("skills/arm/grasp_sensor/limits", [0.0025, 0.18]))
 
+class PublicArm(object):
+    """
+    Public arm interface, also checks a challenge doesn't try to use more than it asked for.
+
+    :ivar _arm: Private link to the real arm.
```

Comment:
Not sure what the convention for docs on these kind of class member variables is

Reply:
It's all allowed and considered to be "variable": http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists